src/language/definition.ts

```diff
@@ -0,0 +1,10 @@
+import { LanguageDefinition } from '@grafana/experimental';
+import { conf, language } from './language';
+
+const timestreamLanguageDefinition: LanguageDefinition & { id: string } = {
+  id: 'timestream',
+  // TODO: Load language using code splitting instead: loader: () => import('./language'),
```

Comment:
should this todo happen now? Why can't it?

Reply:
I don't know it's some weird issue with webpack and I don't even now where to start investigating it. :| Anyway, the language is only about 100 lines of code so not a big issue IMO.